docs/admin/accessRights.rst

```diff
@@ -139,11 +139,32 @@ Programming Exercises
 +--------------------------------+------------+--------+--------------------+
 | Cleanup                        |      ✔     |        |                    |
 +--------------------------------+------------+--------+--------------------+
-| Repository Access              |    R + W   |  R + W |          R         |
-+--------------------------------+------------+--------+--------------------+
 | Build Plan Access              |    R + W   |  R + W |          R         |
 +--------------------------------+------------+--------+--------------------+
 
+Programming Exercises - Repository Access
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**Action Types:** *READ* (fetch, pull), *WRITE* (push), *RESET* (reset to origin head)
+
+| Admins, Instructors and Editors of a course are allowed to *READ*, *WRITE* and *RESET* every repository in their courses.
+| Teaching Assistants are allowed to *READ* every repository in their courses.
+
+The remaining permitted actions for Students and Teaching Assistants on their own repositories are as follows:
+
++--------------------------------------------------------------------+-------------------------------------+
+|                                                                    | Actions                             |
++--------------------------------------------------------------------+-------------------------------------+
+| Before start/ release date                                         | *READ* :sup:`TA` , *WRITE* :sup:`TA`|
```

Comment:
I'd just use "start date" here. The release date is the start date if no start date is set, but this is more an internal technicallity.